src/main/java/MicroRaiden.java

```diff
@@ -39,17 +104,79 @@ public void createAccount(String accountFile) {
         String address = new String(Hex.encodeHex(keyPair.getAddress()));
         System.out.println("Generated new account: 0x" + address);
         byte[] priv = keyPair.getPrivKeyBytes();
-        byte[] nodeid = keyPair.getNodeId();
         
         try {
             OutputStream os = new FileOutputStream(accountFile + ".pkey");
-            os.write(priv);
+            JSONObject obj=new JSONObject();
+            obj.put("privkey", new String(Hex.encodeHex(priv)));
+            obj.put("address", address);
+            os.write(obj.toJSONString().getBytes());
             os.close();
         } catch (IOException e) {
             System.out.println("Couldn't write to file: " + accountFile + " " + e.toString());
         }
     }
     
+    public void createAccountByPrivateKey(String accountFile, String privateKeyHex) {
+    	if(privateKeyHex.startsWith("0x")) {
```

Comment:
indenting is a bit messed up here - also the try catch blocks down a bit farther should have consistent spacing with the other try catch blocks (space between try and { and catch and { for example)